preview/FS-1068-open-type-declaration.md

```diff
@@ -0,0 +1,273 @@
+# F# RFC FS-1068 - Open Type Declaration
+
+The design suggestion [Open static classes](https://github.com/fsharp/fslang-suggestions/issues/383) has been marked "approved in principle".
+This RFC covers the detailed proposal for this suggestion.
+
+
+[Discussion thread](https://github.com/fsharp/fslang-design/issues/352)
+
+* [x] Approved in principle
+* [x] Details: [Resolved to Preview](https://github.com/fsharp/fslang-design/issues/352)
+* [x] Implementation: [Complete to Preview](https://github.com/dotnet/fsharp/pull/6325)
+
+
+## Summary
+[summary]: #summary
+
+Add support for opening types to gain access to static members and nested types, e.g. `open type System.Math`. For example
+
+```fsharp
+> open type System.Math;;
+> Min(1.0, 2.0);;
+val it : float = 1.0
+
+> Min(2.0, 1.0);;
+val it : float = 1.0
+```
+
+## Motivation
+[motivation]: #motivation
+
+This greatly increases the expressivity of F# DSLs by allowing method-API facilities such as named arguments, optional arguments and type-directed overloading to be used in the DSL design.
+
+Type providers can provide static classes, hence this would allow type providers to provide "unqualified" names. This useful for some projections where the natural thing is unqualified, e.g. the "R" type provider always required `R.plot` etc. With this feature, `R.plot` can now be `plot`.
```

Comment:
```suggestion
Type providers can provide static classes, hence this would allow type providers to provide "unqualified" names. This is useful for some projections where the natural thing is unqualified, e.g. the "R" type provider always required `R.plot` etc. With this feature, `R.plot` can now be `plot`.
```